localstack/utils/aws/aws_stack.py

```diff
@@ -418,6 +418,11 @@ def get_s3_hostname():
 
 
 def generate_presigned_url(*args, **kwargs):
+    warnings.warn(
+        "`generate_presigned_url` is obsolete. Use the Boto client `generate_presigned_url` method",
```

Comment:
```suggestion
        "`aws_stack.generate_presigned_url` is obsolete. Use the Boto client `generate_presigned_url` method",
```

Might be a bit clearer, but if that clashes with any other existing deprecation warnings, feel free to ignore.

Reply:
Good idea :+1: